docs/DeliverabilityAnalysis.md

```diff
@@ -24,6 +30,12 @@ instance = Lob::DeliverabilityAnalysis.new(
   dpv_cmra: null,
   dpv_vacant: null,
   dpv_active: null,
+  dpv_inactive_reason: null,
```

Comment:
Need to update nulls to nil here. `null` in Ruby is `nil`
```suggestion
  dpv_inactive_reason: nil,
```

Reply:
Updated the docs I updated here, but I'll set a reminder to update this in the sdk generator repo

Reply:
Fixed the docs I updated here, but I'll set a reminder to update this in the sdk generator repo